src/components/web3Connection/Account/Account.tsx

```diff
@@ -2,97 +2,165 @@ import {
   Button,
   ButtonGroup,
   Divider,
+  HStack,
+  Text,
   useBreakpointValue,
+  useColorMode,
   useDisclosure,
+  VStack,
 } from "@chakra-ui/react"
 import { UnsupportedChainIdError, useWeb3React } from "@web3-react/core"
 import Card from "components/common/Card"
 import { useCommunity } from "components/community/Context"
 import { Web3Connection } from "components/web3Connection/Web3ConnectionManager"
+import { Chains } from "connectors"
 import useBalance from "hooks/useBalance"
-import { LinkBreak, SignIn, Wallet } from "phosphor-react"
+import { LinkBreak, SignIn } from "phosphor-react"
 import { useContext } from "react"
 import type { Token } from "temporaryData/types"
 import shortenHex from "utils/shortenHex"
 import AccountModal from "../AccountModal"
+import Identicon from "../components/Identicon"
 import useENSName from "./hooks/useENSName"
 
 type Props = {
   token: Token
 }
 
+const AccountCard = ({ children }): JSX.Element => {
```

Comment:
It'd be better to just use isMobile for conditional rendering, render a Card on desktop as before and the customized Card on mobile